README.md

```diff
@@ -1,8 +1,12 @@
 # pre
 
+[docs_master_badge]: https://img.shields.io/badge/docs.rs-master-green
+[docs_master_url]: https://aticu.github.io/pre
```

Comment:
You'd have to change the `docs_master_url` if you have already a personal GitHub Pages website that points to a custom URL using `CNAME` or similar. 
Take a look at https://raw.githubusercontent.com/SirWindfield/twm/master/README.md, which does exactly this.